Backtesting/collectors/exchanges/exchange_history_collector.py

```diff
@@ -88,6 +88,7 @@ def _load_all_available_timeframes(self):
 
     async def get_ohlcv_history(self, exchange, symbol, time_frame):
         candles: list = await self.exchange.get_symbol_prices(symbol, time_frame)
+        self.exchange_manager.uniformize_candles_if_necessary(candles)
```

Comment:
Do you think this is mandatory ?

Reply:
to keep second level timestamps, we either do it once here or all the time at import

Reply:
Currently, backtesting candles only come from a subscription on OHCLV_CHANNEL that already use `uniformize_candles_if_necessary` before notifying its consumer.

Reply:
not for exchange_history_collector.py, it calls the rest_exchange directly (so that users can select pairs that are not currently watched in channels)

Reply:
Oh yes you are right :+1: